Cargo.toml

```diff
@@ -23,6 +23,7 @@ serde = { version = "1.0", features = ["derive"] }
 static_assertions = "1.1.0"
 subtle = "2.4.1"
 thiserror = "1.0"
+zipf = "7.0.0" # XXX Figure out how to fence this dependency for benchmarking
```

Comment:
Reviewer note: I'm not sure how to fence this dependency to benchmarks. We certainly don't want it as a main dependency.

Reply:
dev-dependency should be good, as benchmarks get built like a different flavor of integration tests